plugins/module_utils/ipa.py

```diff
@@ -83,7 +83,7 @@ def login(self, username, password):
                 if status_code not in [200, 201, 204]:
                     self._fail('login', info['msg'])
 
-                self.headers = {'Cookie': resp.info().get('Set-Cookie')}
+                self.headers = {'Cookie': info['set-cookie']}
```

Comment:
Is `set-cookies` always there? If not, this code will now crash. (Maybe use `get()` instead of `[]`?)

Reply:
set-cookies will always be there because ipa api login endpoint replies the session token in this header.